knex/seeds/01_users.js

```diff
@@ -0,0 +1,17 @@
+
+exports.seed = function (knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('users').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('users').insert([
+        {
+          id: 1,
+          firstname: 'Eunice',
+          lastname: 'Min',
+          email: 'euniceem1007@gmail.com',
```

Comment:
You should keep the password here but you'll need to add bcrypt later if you want this user seed account to work when you have authentication setup on your server.